### PR TITLE
PROGRAMMES-6457 Composer update

### PR DESCRIPTION
**Pull request by votemike**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6457

Because symfony/phpunit-bridge is the thing that installs PHPUnit, PHPUnit has to remain on version 6 (even though version 7 is out). This means that we can't yet update johnkary/phpunit-speedtrap, phpstan/phpstan or phpstan/phpstan-phpunit to their latest versions.

GitHub hides the diff of composer.lock by default, but obviously you should make it show and check it.

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by BjornTwachtmann** [comment]
Tests are failing
```There was 1 failure:

1) Tests\App\Controller\Schedules\HomeControllerTest::testController
Failed asserting that the response status code "500" matches expected "200".
Failed asserting that 500 matches expected 200.```

**Comment by BjornTwachtmann** [comment]
Also, we really want to update all the symfony stuff to 3.4.12, which this PR doesn't seem to do? I think it should be possible

**Comment by votemike** [comment]
Tests are failing because something in PPS needs updating.
I can update Symfony too if you like, I was leaving it for the next ticket (https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6455)

File: composer.json [hunk 1]
```diff
@@ -29,19 +29,21 @@
...
+        "doctrine/orm": "2.6.*",
         "incenteev/composer-parameter-handler": "^2.1",
+        "sensiolabs/security-checker": "^4.1",
         "stof/doctrine-extensions-bundle": "^1.2",
         "symfony/asset": "^3.4",
         "symfony/console": "^3.4",
         "symfony/debug-bundle": "^3.4",
         "symfony/flex": "^1.0",
         "symfony/framework-bundle": "^3.4",
+        "symfony/http-kernel": "v3.4.4",
```
Comment (votemike) [inline composer.json]: I've had to lock the at 3.4.4 as 3.4.5 breaks everything 👎 
Anyone know how to fix it?

Comment (votemike) [reply composer.json]: If you run `composer remove symfony/http-kernel`, you will see the problem.

**Comment by votemike** [comment]
Test this please

**Comment by votemike** [comment]
Test this please

**Comment by votemike** [comment]
Test this please

**Pull request merged**